optimize.js

```diff
@@ -0,0 +1,79 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+async function viewDashboard([dashboardId]) {
+
+const accessToken = await getAccessToken();
+const optimizeApiUrl = process.env.OPTIMIZE_API_URL;
+const collectionId = process.env.OPTIMIZE_COLLECTION_ID;
```

Comment:
Blocking (I think): I think I'd expect this collection Id to come from the command arguments, rather than an environment variable. This would allow the CLI to be used for any optimize collections.